src/plotman/job.py

```diff
@@ -209,6 +204,8 @@ def get_running_jobs(
             # both identified as plot processes.  Only the child is
             # really plotting.  Filter out the parent.
 
+            pids = {process.pid for process in processes}
+            ppids = {process.ppid() for process in processes}
```

Comment:
Are these (and above and below) needed?  Or perhaps they are just some confusion during some merges that can be removed?

Reply:
Those must have been something I overlooked during the merges, there shouldn't have been any changes made to any files except plotman.py and graph.py. I dug a little deeper and it looks like several other files were changed, I'm not totally sure how, but I've updated them all to reflect the current state of the development branch, sorry about that.